src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -311,4 +312,37 @@ public function theAdministratorShouldSeeThatThisOrderHasBeenPlacedIn(
 
         Assert::same($currencyCode, $currency);
     }
+
+    /**
+     * @Then I should see an order with :orderNumber number
+     */
+    public function iShouldSeeOrderWithNumber(string $orderNumber)
+    {
+        $response = $this->client->getLastResponse();
+
+        Assert::true(
+            $this->responseChecker->hasItemWithValue($response, 'number', $orderNumber),
+            sprintf('No order with number "%s" has been found.', $orderNumber),
+        );
+    }
+
+    /**
+     * @When I switch the way orders are sorted by :fieldName
```

Comment:
It would be good to keep the order of the methods: `Given` -> `When` -> `Then`, so this one should be moved before `Then` method, which is above